docs/Project.toml

```
@@ -2,4 +2,4 @@
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 
 [compat]
-Documenter = "0.27"
+Documenter = "0.27, 1"
```

Maybe we could simply require version 1? There doesn't seem to be any advantage in supporting 0.27, anyway this is only used to build docs.
```suggestion
Documenter = "1"
```